.github/workflows/codeql.yml

```
@@ -1,17 +1,101 @@
-      java-build-paths: "['./refarch-backend','./refarch-eai']"
+  check-changes:
+    name: Check changed files and directories
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: read
+    outputs:
+      java: ${{ steps.filter.outputs.java }}
+      javascript-typescript-vue: ${{ steps.filter.outputs.javascript-typescript-vue }}
+      python: ${{ steps.filter.outputs.python }}
+    steps:
+      - name: Checkout repository
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 10 but found 6

(indentation)